src/DTO/DefaultProxyFeature.php

```diff
@@ -28,7 +28,8 @@
      *             value: string
      *         }
      *     },
-     *     impression_data: bool
+     *     impression_data?: bool,
+     *     impressionData?: bool
```

Comment:
One will be set, the other won't, we don't know which one because it depends on the version of the backend